docs/source/markdown/podman-run.1.md

```diff
@@ -674,11 +674,10 @@ Restart policy will not take effect if a container is stopped via the **podman k
 
 Valid _policy_ values are:
 
-- **no**: Do not restart containers on exit;
-- **on-failure**[:*max_retries*]: Restart containers when they exit
-with a non-zero exit code, retrying indefinitely or until the optional
-*max_retries* count is hit;
-- **always**: Restart containers when they exit, regardless of status, retrying indefinitely.
+- `no`                       : Do not restart containers on exit
+- `on-failure[:max_retries]` : Restart containers when they exit with a non-zero exit code, retrying indefinitely or until the optional *max_retries* count is hit
+- `always`                   : Restart containers when they exit, regardless of status, retrying indefinitely
+- `unless-stopped`           : Identical to **always**
```

Comment:
the code comment below says it's not identical to always....  If unless-stopped is not supported at the moment, I'd just remove it from the man pages until it is.